StyleFix

asw-234 opened the pull request:
My "**_corrected_**" version of the program. Hope it isn't that painful to look at :^)

Nightyfire committed:
Declared cout, cin and endl from namespace std.

Cout, cin and endl are declared in the beginning of the program with the
help of the 'using' command, so as not to write "std::" every time
those commands are needed.

Nightyfire committed:
Fixed indentation.

+ Added or removed whitespace depending on the situation.

Nightyfire committed:
Grouped arrays into a structure named Pesni.

Nightyfire committed:
Version 0.1

+ Extracted some repeating code into functions
	(menu, sortSongs, printOrder,
	printCurrentSongDetails, currentSongPosition)
  mainly for main() cases 3, 4 and 11.

+ Rewrote some of the code in English (variables, function
  parameters etc.)

+ Renamed structure Pesni into SongManagement

+ Changed curly brackets style

+ Minor output adjustments

Nightyfire committed:
Version 0.2

+ Extracted repeating code in case 5 and 6 of main(), putting it into a
function called "search"

+ Minor output adjustments

+ Added some comments

        *Upcoming: variable name fix, more repeating code extraction,
		   remake of function sortSongs*

Nightyfire committed:
Version 0.3

+ Added function printPlaylist that prints out the current playlist

+ Replaced repeating code in main() case 7 and 8

+ Minor style fixes (curly braces, spaces)

Nightyfire committed:
Version 0.4

+ Extracted repeating code inside of function sortSongs
  into 2 functions: charMoveUpByOne and intMoveUpByOne

+ Repositioned some variables/constants at the start of main()

+ realnoe_kolichestvo_pesen: changed variable's name to "const int"

+ Minor style fixes (curly braces, spaces)

+ Minor output adjustments

+ Added some comments

Nightyfire committed:
Version 0.5

+ Added struct SongManagement members:
  "pesny_VTope","gruppy_v_tope", "popularnost_pesen_v_tope"

+ Extracted repeating code in main() case 10 and 12 into a function named
  sortTopSongs

+ Generalised function printOrder. Made adjustments likewise

+ Removed duplicate function printPlaylist

+ Replaced repeating code with a function in main() case 10

+ Minor output changes

+ Minor style fixes (spaces)

Nightyfire committed:
Version 0.6

+ Rewrote the whole code into English

Nightyfire committed:
Version 0.7

+ Gave a more acceptable name to constant "numberOfsongs"

+ Removed obsolete constant "MAX_SONGS_NUMBER". Replaced it with
  "NUMBER_OF_SONGS"

+ Minor style fixes (spaces)

Nightyfire committed:
Version 0.7.1

+ Minor style fixes

Nightyfire committed:
Version 0.7.2

+ Renamed bool variable "nashel" to its english equivalent

+ Minor style fixes

Nightyfire committed:
Version 0.7.3

+ Fixed indentation for github code display (replaced all tabs with 4 spaces)

g4rfan commented on main.cpp:
```diff
@@ -1,469 +1,512 @@
+    const char *band;
+    int popularity;
+
+    const char *topSongs;
```
в этой структуре дублирующиеся по смыслу поля

Nightyfire committed:
Version 0.8

+ Removed duplicate structure members

+ Removed duplicate function sortTopSongs

+ Removed duplicate action in function printOrder

+ Adjusted the rest of the code accordingly

+ Minor output changes

Nightyfire committed:
Version 0.9

+ Split up the code into different header and source code files.

  New header files: Printing.h, Search.h, Sorting.h,
  SongManagementStruct.h

  New source code files: Printing.cpp, Search.cpp, Sorting.cpp

+ Changed main.cpp file according to the changes

+ Minor output fixes

Nightyfire committed:
Version 0.9.1

+ Removed a commented out line in SongManagementStruct header
